[2025-04-12T16:09:36.349+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weekly_comment_analysis_pipeline.run_bertopic manual__2025-04-12T16:04:13.520840+00:00 [queued]>
[2025-04-12T16:09:36.354+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weekly_comment_analysis_pipeline.run_bertopic manual__2025-04-12T16:04:13.520840+00:00 [queued]>
[2025-04-12T16:09:36.354+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-04-12T16:09:36.362+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_bertopic> on 2025-04-12 16:04:13.520840+00:00
[2025-04-12T16:09:36.367+0000] {standard_task_runner.py:60} INFO - Started process 48625 to run task
[2025-04-12T16:09:36.371+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weekly_comment_analysis_pipeline', 'run_bertopic', 'manual__2025-04-12T16:04:13.520840+00:00', '--job-id', '264', '--raw', '--subdir', 'DAGS_FOLDER/Sentiment_Topicmodelling.py', '--cfg-path', '/tmp/tmp05b83nko']
[2025-04-12T16:09:36.373+0000] {standard_task_runner.py:88} INFO - Job 264: Subtask run_bertopic
[2025-04-12T16:09:36.417+0000] {task_command.py:423} INFO - Running <TaskInstance: weekly_comment_analysis_pipeline.run_bertopic manual__2025-04-12T16:04:13.520840+00:00 [running]> on host b22c3b1dd6e4
[2025-04-12T16:09:36.475+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weekly_comment_analysis_pipeline' AIRFLOW_CTX_TASK_ID='run_bertopic' AIRFLOW_CTX_EXECUTION_DATE='2025-04-12T16:04:13.520840+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-12T16:04:13.520840+00:00'
[2025-04-12T16:09:36.541+0000] {SentenceTransformer.py:208} INFO - Use pytorch device_name: cpu
[2025-04-12T16:09:36.542+0000] {SentenceTransformer.py:216} INFO - Load pretrained SentenceTransformer: sentence-transformers/all-MiniLM-L6-v2
[2025-04-12T16:09:46.479+0000] {local_task_job_runner.py:302} WARNING - State of this instance has been externally set to success. Terminating instance.
[2025-04-12T16:09:46.481+0000] {process_utils.py:131} INFO - Sending 15 to group 48625. PIDs of all processes in the group: [48625]
[2025-04-12T16:09:46.482+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 48625
[2025-04-12T16:09:46.484+0000] {taskinstance.py:2450} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-04-12T16:09:46.514+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=48625, status='terminated', exitcode=0, started='16:09:35') (48625) terminated with exit code 0
